From 37fa0b13a2503039ea0e9ba785f1951785d941ad Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Mon, 27 Feb 2017 19:45:54 +0000
Subject: [PATCH] OS-5565 Need smrt HBA mode support Reviewed by: Joshua M.
 Clulow <jmc@joyent.com> Approved by: Joshua M. Clulow <jmc@joyent.com>

---
 overlay/generic/etc/driver_classes | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/overlay/generic/etc/driver_classes b/overlay/generic/etc/driver_classes
index 8608747a..3124f9a2 100644
--- a/overlay/generic/etc/driver_classes
+++ b/overlay/generic/etc/driver_classes
@@ -27,4 +27,4 @@ qlc	fibre-channel
 si3124	scsi-self-identifying
 dr_sas	scsi-self-identifying
 skd	disk
-smrt	scsi
+smrt	scsi-self-identifying
-- 
2.21.0

